src/Samples/Sample.AspNetCore/Controllers/CheckOutController.cs

```diff
@@ -84,13 +80,13 @@ public async Task<PaymentOrder> CreatePaymentOrder(string consumerProfileRef = n
             }
         }
 
-        public async Task<SwedbankPay.Sdk.Payments.Card.Payment> CreateCardPayment()
+        public async Task<SwedbankPay.Sdk.Payments.CardPayments.CardPayment> CreateCardPayment()
```

Comment:
I suppose it's an easy task to remove all these superfluous namespaces with ReSharper once this PR is merged, @zunkas?

Reply:
Move everything to Payments instead? 
Should be easy